tests/test_other.py

```diff
@@ -798,7 +798,10 @@ def verify_includes(stderr):
       includes = stderr[start:end]
       includes = [i.strip() for i in includes.splitlines()[1:-1]]
       for i in includes:
-        self.assertContained(path_from_root('system'), i)
+        if shared.Cache.dirname in i:
```

Comment:
this isn't clear to me. perhaps a comment?

Reply:
Sorry this isn't best test in the world.  Adding a comment.